src/RadixDlt.NetworkGateway.PrometheusIntegration/DataAggregatorMetricsObserver.cs

```diff
@@ -573,4 +579,9 @@ ValueTask ITransactionStreamReaderObserver.GetTransactionsFailed(string nodeName
 
         return ValueTask.CompletedTask;
     }
+
+    public void OnSqlQueryExecuted(string queryName, TimeSpan duration)
+    {
+        _sqlQueryDuration.WithLabels(queryName).Observe(duration.TotalMilliseconds);
```

Comment:
I see we're using default histogram buckets, therefore we should use `duration.TotalSeconds` over here.